docs/sections/qualifiers.md

```diff
@@ -1,39 +1,47 @@
+Overlay qualifiers refine/qualify when an overlay is applied to a YAML document within a YAML file path.  
 
+The Yaml Overlay Tool `Yot` has two kinds of qualifiers that can be used together, separately, or not at all.
 
-### documentQuery Overlay Qualifier
+* `documentQuery`
+* `documentIndex` 
 
-The `documentQuery` qualifier can be used on either `commonOverlays` or the `overlays` key on a `yamlFiles.path`, but cannot be used under the `documents` key.  The purpose of a `documentQuery` is to qualify an overlay operation by checking for a value or multiple values contained in a YAML document within a file.
 
-Think of the `documentQuery` as groups of conditions that must be met before applying this overlay to the YAML document. Only one group of `conditions` must all match prior to qualifying the application of an overlay.
+### documentQuery overlay qualifier
+
+The `documentQuery` qualifier can be used on either `commonOverlays` or the `overlays` key on a `yamlFiles.path`. It cannot be used under the `documents` key.  The purpose of a `documentQuery` is to qualify an overlay operation by checking for a value or multiple values contained in a YAML document within a file.
+
+The `documentQuery` results include groups of conditions that must be met before applying this overlay to the YAML document. Only one group of `conditions` must all match prior to qualifying the application of an overlay.
 
 The `documentQuery` key is a list/array which contains a list of the following top-level keys:
 
+* documentQuery Top-level
+* documentQuery conditions
+
 
-#### documentQuery Top-level keys
+#### documentQuery top-level keys
 
-| Key | Description | Type |
+| **Key** | **Description** | **Type** |
 | --- | --- | --- |
-| conditions | A grouping of conditions that must exist in a YAML document to qualify application of an overlay. Each `documentQuery` list/array can contain one or many `conditions` groups.  Each list/array of `conditions` contains a list/array of key/value pairs that must all return valid matches with expected values prior to qualifying application of an overlay. Each group of `conditions` is treated as an implicit "or", while the key/value conditions in each grouping is treated as an implicit "and". | list/array |
+| Conditions | A group of conditions that must exist in a YAML document to qualify application of an overlay. Each `documentQuery` list/array can contain one or more `conditions` groups.  Each list/array of `conditions` contain a list/array of key/value pairs that must all return valid matches with expected values prior to qualifying the application of an overlay. Each group of `conditions` is treated as an implicit "or", while the key/value conditions in each grouping is treated as an implicit "and". | List/array |
```

Comment:
@lizz7000 these are actual values that need to be lower-cased in the API.  Do you mind flipping the conditions "key" back?